services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -141,7 +149,7 @@ const sx = {
   deleteButton: {
     padding: "0",
     background: "white",
-    "&:hover": {
+    "&:hover:disabled": {
       background: "white",
     },
```

Comment:
Question: Do we want this behavior _only_ when hovering _and_ disabled, or both when hovering _and_ when hovering + disabled?

Reply:
_applies to other locations in PR_

Reply:
Both!